src/instance.ts

```diff
@@ -16,13 +16,101 @@
 
 import arrify = require('arrify');
 const common = require('@google-cloud/common-grpc');
-import {ServiceObjectConfig} from '@google-cloud/common';
+import {ServiceObjectConfig, GetConfig} from '@google-cloud/common';
 import {paginator} from '@google-cloud/paginator';
 import {promisifyAll} from '@google-cloud/promisify';
 import * as extend from 'extend';
-import * as is from 'is';
 import snakeCase = require('lodash.snakecase');
 import {Database} from './database';
+import {google as instanceAdmin} from '../proto/spanner_instance_admin';
+import {google as databaseAdmin} from '../proto/spanner_database_admin';
+import {Operation as GaxOperation} from 'google-gax';
+import {
+  SessionPoolOptions,
+  SessionPoolInterface,
+  SessionPool,
+} from './session-pool';
+import {Spanner} from '.';
+import {ServiceError} from 'grpc';
+import {RequestCallback, PagedResponse} from './common';
+import {Stream} from 'stream';
+
+export type IDatabase = databaseAdmin.spanner.admin.database.v1.IDatabase;
+export type IInstance = instanceAdmin.spanner.admin.instance.v1.IInstance;
+export type ILongrunningOperation = instanceAdmin.longrunning.IOperation;
```

Comment:
Any reason we don't just call this `IOperation`?

Reply:
change `ILongrunningOperation` to `IOperation`.